pkg/kubelet/certificate/certificate_store_test.go

```diff
@@ -278,7 +280,9 @@ func TestLoadFile(t *testing.T) {
 	}()
 
 	pairFile := filepath.Join(dir, "kubelet-pair.pem")
-	if err := ioutil.WriteFile(pairFile, []byte(certificateData+"\n"+privateKeyData), 0600); err != nil {
+	data := append(testCertificates[0].certificatePEM, []byte("\n")...)
+	data = append(data, testCertificates[0].keyPEM...)
```

Comment:
In fact, once I use `...` on the last parameter, the compiler won't permit anything in the middle. https://play.golang.org/p/Xg-z__MhW1